nautobot/core/templates/generic/object_create.html

```diff
@@ -50,7 +50,7 @@ <h3>
             </div>
         </div>
         <div class="row">
-            <div class="col-md-6 col-md-offset-3 text-right">
+            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1 text-right">
```

Comment:
Are there model-specific templates that may also need updates? Grepping for `col-md-offset-3 text-right` shows a few candidates such as `ipam/ipaddress_assign.html` and `virtualization/cluster_add_devices.html`.

Reply:
Looks like those were missed in #5313 so the bug wouldn't be present here. Do we want to make the #5313 changes to those two templates?

Reply:
It would be good to update them for consistency's sake but I'm also fine if you want to consider that out of scope for this PR.